src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -43,11 +44,12 @@ public static IEnumerable<IPackage> GetPackages(ChocolateyConfiguration configur
                 results = results.Where(p => p.IsLatestVersion);
             }
 
-            return results.OrderBy(p => p.Id)
-                          .AsEnumerable()
-                          .Where(PackageExtensions.IsListed)
-                          .Where(p => configuration.Prerelease || p.IsReleaseVersion())
-                          .distinct_last(PackageEqualityComparer.Id, PackageComparer.Version).ToList();

```

Comment:
Well, I'm pretty sure you can do the same thing with the sort/group/first, but the output appears to be correct without the distinct_last and I honestly think it would be a big problem if more than one item in the feed was marked as `IsLatestVersion` so I don't see the point ;-)

Maybe the original (nuget) code was just paranoid...